bin/travis_analyzer

```diff
@@ -13,6 +17,7 @@ class TravisAnalyzer
 
     @upgrade_analyzer = UpgradeAnalyzer::Analyzer.new(@travis, repo_name, github_token)
     @log_analyzer = LogAnalyzer::Analyzer.new(@travis, repo_name, github_token)
+    @knapsack_analyzer = KnapsackAnalyzer::Analyzer.new(@travis, repo_name, github_token)
```

Comment:
I think this should not be in this commit since the class doesn't exist yet.

Reply:
fixed